scipy/stats/_stats_py.py

```diff
@@ -4363,6 +4371,10 @@ def confidence_interval(self, confidence_level=0.95, method=None):
         return ci
 
 
+xp_capabilities_table['pearsonr'] = dict(cpu_only=True, exceptions=['cupy'])
+
+
+@xp_capabilities()
```

Comment:
UX nit: the `@xp_capabilities` decorator could inject into `xp_capabilities_table` itself
```suggestion
@xp_capabilities(cpu_only=True, exceptions=['cupy'])
```

Reply:
I actually described that it would work that way before writing it. Not sure why I didn't do that, so I'll change it.